scripts/release/createWebModules.js

```diff
@@ -104,7 +104,7 @@ async function createAtlasWebContentModule() {
 }
 
 async function createAtlasCoreModule() {
-    console.log("Creating the Atlas Web Content module.");
+    console.log("Creating the Atlas Web Core module.");
```

Comment:
Should we remove "Web" ?

Reply:
Changed. Although the file (createWebModules) and github action (createWebRelease) focuses on web. I suppose this is okay, so long as it's documented.